Launcher/Launcher/src/Helpers/VersionHelper.cs

```diff
@@ -0,0 +1,60 @@
+using System.Reflection;
+using System.Text.Json;
+using WSL;
+
+namespace Launcher.Helpers;
+
+public static class VersionHelper {
+    public static Version? LauncherVersion => Assembly.GetEntryAssembly()?.GetName().Version;
```

Comment:
```suggestion
    public static Version? LauncherVersion => type(VersionHelper).Assembly.GetName().Version;
```

see https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getexecutingassembly?view=net-8.0&redirectedfrom=MSDN#remarks